### ISSUE TITLE
Feature Request : Keep lost clients 

### ISSUE THREAD

**Issue by CloudyWuff** [cloudradar-monitoring/rport#426]
Add an option to keep disconnected clients until manualy removed via console or frontend

**Comment by thorstenkramm** [comment]
@CloudyWuff  You can set the[ `keep_lost_clients`](https://github.com/cloudradar-monitoring/rport/blob/master/rportd.example.conf#L110) settings in rportd.conf to the maximum of 168h (=7 days).
Isn't that not enough?
Manual removal of disconnected clients is already there. 

![image](https://user-images.githubusercontent.com/16852023/150178933-06b60239-3a26-4116-b2a4-71be283f1cde.png)

**Comment by werwarfri** [comment]
> Add an option to keep disconnected clients until manualy removed via console or frontend

this would be a good feature 👍

**Comment by CloudyWuff** [comment]
@thorstenkramm Thanks for your reply
I already set this setting in the rportd.conf but i'd like to deactivate the automatic removal of the clients. I want to prevent this for devices like 3cx SBC since we pre configure those and store until needed. This way i can see in rport how many devices are stored. Im sure there are more use cases i can't think of right now. Also it would be nice if we can store the monitoring data indefinitely as im not sure if i can set this at the moment. There is a "data_storage_days = x" setting but im not sure if it can be disabled.

**Comment by thorstenkramm** [comment]
It's queued in our task system. Soon you will be able to turn the cleanup routine off completely.

**Comment by kevs-oc** [comment]
I'm currently using v0.5.0 and found the following in the conf file:
`  ## An optional param to define a duration to keep info (clients, tunnels, etc) about active and disconnected clients.
  ## By default is "1h". To disable it set it to "0". It can contain "h"(hours), "m"(minutes), "s"(seconds).
  keep_lost_clients = "0"
`
It was originally set to 168 but I had disconnected clients disappearing from the list. I set the value to 0 and restarted the rportd service. Now all disconnected clients are gone from the list!

Is there any way to get them back? What happens when a client that has been dropped from the list tries to connect to the server?

**Comment by kevs-oc** [comment]
The description is somewhat misleading on the keep_lost_clients param. It says to set it to 0 to disable it, which I interpreted to mean that whatever process removed the lost clients would not fire. Instead it means to not keep ANY lost client information at all.

I second @CloudyWuff request to keep all configured clients in the list unless they have been deleted via the GUI.

For anyone looking, the clients that have been removed from the list can still connect back to the server and show up online. I thought I had stranded them.

**Comment by thorstenkramm** [comment]
@kevs-oc 
You are right. We already changed the comment in the current version of the [rportd.conf](https://github.com/cloudradar-monitoring/rport/blob/master/rportd.example.conf#L110-L114).
`0` means, disconnected clients are purged immediately. It disables keeping them. 
In future releases, we provide the option to turn the housekeeping off completely, so disconnected clients remain there forever.

**Comment by thorstenkramm** [comment]
Solved with [version 0.6.3](https://github.com/cloudradar-monitoring/rport/releases/tag/0.6.3)
You can disable the housekeeper and keep disconnected clients forever. 

New config lines for your `rportd.conf`:
```
## An optional param to define whether disconnected clients get cleaned up.
## By default the clients are cleaned up.
#cleanup_lost_clients = true
```

0.6.3 is compatible with the 0.6.0 frontend. You are welcome to test the pre-release.
